src/components/Navigation/Toolbar/Toolbar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import '../../../assets/semantic/semantic.min.css';
+import { Menu } from 'semantic-ui-react';
+
+const toolbar = (props) => (
+    <Menu>
+        <Menu.Item>
+            <Link to="/" >Home</Link>
```

Comment:
use map to render the menu items
take const and reader menus using map
optimize